package.json

```diff
@@ -92,6 +92,7 @@
     "@types/json-schema": "^7.0.9",
     "@types/lodash.clonedeep": "^4.5.6",
     "@types/lodash.isequal": "^4.5.5",
+    "@types/mock-express-request": "^0.2.0",
```

Comment:
I set back the **mock-express-request** because we need to set the request body as a string for the write endpoint and this is not doable with **node-mocks-http**

Reply:
Oh crap!
I see, thanks.